### PR TITLE
 Update to CakePHP 3.7.x

### LINKED ISSUE #17: Update to CakePHP 3.7.x
Update CakePHP to version 3.7.x.

### PR DESCRIPTION
**Pull request by TekkCraft**
Unfinished.

Closes #17

### CODE REVIEW

File: docs/Sensors.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ Checks whether a connection to the database server can be established.
 #### DB Up to date
 Uses the Migrations Plugin to check whether all migrations have been run.
 
+This sensor requires you to either load the cakephp/migrations plugin before the sensor is called or to use cakephp/migrations:^2.2 once released.
```
Comment (ravage84) [inline docs/Sensors.md:10]: We aren't yet sure which version it will be :thinking:

**Review by liviakuenzli — APPROVED** [review]

**Review by ravage84 — COMMENTED** [review]
We should wait for cakephp/migrations to release a new version.

File: docs/Sensors.md (hunk 1 above)
Comment (ravage84) [reply docs/Sensors.md:10]: It's version [2.1,1](https://github.com/cakephp/migrations/releases/tag/2.1.1) now.

File: src/Heartbeat/Sensor/DBUpToDate.php [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
-use Cake\Core\Plugin;
 use Migrations\Migrations;
 use OrcaServices\Heartbeat\Heartbeat\Sensor;
 
 /**
  * DB Up to Date Sensor
  *
- * This sensor depends on the CakePHP Migrations plugin
+ * This sensor depends on the CakePHP Migrations plugin.
+ * Make sure the plugin is loaded before calling this sensor.
```
Comment (ravage84) [inline src/Heartbeat/Sensor/DBUpToDate.php:12]: It shouldn't be necessary with version [2.1,1](https://github.com/cakephp/migrations/releases/tag/2.1.1) of the plugin.

**Comment by ravage84** [comment]
We should suggest the cakephp/migrations plugin in version 2.1.1+ in the composer.json. Analogous to the CakePHP 2.x compatible version.

**Pull request closed without merging**